tests/test_simulation.py

```diff
@@ -865,6 +855,73 @@ def test_noises_digital(matrices, noise, result, n_collapse_ops, seq_digital):
         seq_digital.register.qubits
     )
     trace_2 = res.states[-1] ** 2
+    assert np.abs(np.trace(trace_2)) < 1 and not np.isclose(
```

Comment:
Was the `np.abs()` needed because there was an imaginary part?

Reply:
It was not really needed, but just at some point I had some issues with this test and I was wondering if it was coming from there...

Reply:
Ok, let's revert it then!